docs/antrea-proxy.md

```diff
@@ -253,6 +253,24 @@ data:
       proxyLoadBalancerIPs: false
 ```
 
+It's worth noting that AntreaProxy will bypass all the external `loadBalancerIP`s
+with the above configuration. After K8s v1.29, field `LoadBalancerIPMode` was
+introduced to `LoadBalancerIngress` in Service status, providing users with a
```

Comment:
What's `LoadBalancerIngress`? I don't think it's the name of a field or a feature.

Reply:
You could find it in this link https://github.com/kubernetes/kubernetes/blob/3f7a50f38688eb332e2a1b013678c6435d539ae6/staging/src/k8s.io/api/core/v1/types.go#L4867. I will rephrase the sentences containing the word `LoadBalancerIngress` that confuses readers.